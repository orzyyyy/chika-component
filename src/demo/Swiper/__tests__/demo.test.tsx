import React from 'react';
import { mount } from 'enzyme';
import 'nino-cli/scripts/setup';
import SwiperDemo from '..';

describe('SwiperDemo', () => {
  it('render correctly', () => {
    const wrapper = mount(<SwiperDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
