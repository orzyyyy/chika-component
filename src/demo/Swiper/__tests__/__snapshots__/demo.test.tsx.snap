// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwiperDemo render correctly 1`] = `
<SwiperDemo>
  <div
    className="Swiper_demo"
  >
    <Swiper
      duration={0.7}
      loading={true}
      onLoad={[Function]}
      onRefresh={[Function]}
      sensibility={1}
      wrapperHeight={2950}
    >
      <div
        className="Swiper"
      >
        <div
          className="wrapper"
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "height": 2950,
                "transform": "translate3d(0px, 0px, 0)",
              }
            }
          >
            <ul />
          </div>
          <div
            className="refresh"
            style={
              Object {
                "transform": "translate3d(0px, 0px, 0)",
              }
            }
          >
            <span
              className="refresh-icon"
              style={
                Object {
                  "transform": "rotateZ(0deg)",
                }
              }
            >
              <img
                src="./assets/down.png"
              />
            </span>
            <span
              className="refresh-text"
            >
              下拉刷新
            </span>
          </div>
          <div
            className="load"
            style={
              Object {
                "transform": "translate3d(0px, 0px, 0)",
              }
            }
          >
            <span
              className="load-icon"
              style={
                Object {
                  "transform": "rotateZ(0deg)",
                }
              }
            >
              <img
                src="./assets/down.png"
              />
            </span>
            <span
              className="load-text"
            >
              加载更多
            </span>
          </div>
        </div>
      </div>
    </Swiper>
  </div>
</SwiperDemo>
`;
